test/javascript/components/Editor/features/keyboard-shortcuts.test.js

```diff
@@ -0,0 +1,19 @@
+jest.mock('../../../../../app/javascript/components/editor/FileEditor')
```

Comment:
I decided to break up the `Editor.test.js` into smaller parts. It was getting too hard just to keep on adding tests to it.